Tests/LibWeb/Text/input/css/initially-paused-animation.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<style>
+    body {
+        margin: 0;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        background-color: #f0f0f0;
+    }
```

Comment:
This is a bit of a nitpick, but this styling doesn't seem necessary for the test. Getting rid of it also makes it esasier to see the text output.